Dockerfile

```diff
@@ -14,8 +14,60 @@ RUN useradd -u 663 release
 RUN apt-get update && apt-get install -y python python-pip
 RUN pip install pyparsing && pip install -i https://pypi.wikia-services.com/simple/ wikia.crowdin
 
-# prepare build dir
-RUN mkdir -p /build
-COPY package.json /build
-COPY . /build
-WORKDIR /build
+# create folder to cache dependencies
+WORKDIR /app
+COPY package.json .
+COPY bower.json .
+
+# install npm globals
+RUN npm install -g bower
+RUN npm install -g ember-cli
```

Comment:
These two shouldn't be required as we don't run them directly but through npm scripts which use the local version anyway.